src/mumble/NetworkConfig.ui

```diff
@@ -296,6 +296,29 @@ Prevents the client from downloading images embedded into chat messages with the
     </widget>
    </item>
    <item>
+    <widget class="QGroupBox" name="qgbPrivacy">
+     <property name="title">
+      <string>Privacy</string>
+     </property>
+     <layout class="QVBoxLayout" name="verticalLayout_4">
+      <item>
+       <widget class="QCheckBox" name="qcbHideOS">
+        <property name="toolTip">
+         <string>Prevent OS information being sent to the server</string>
+        </property>
+        <property name="whatsThis">
+         <string>&lt;b&gt;Don't send OS information to the server&lt;/b&gt;&lt;br/&gt;
+Prevents the client from sending potentially identifying information about the operating system to the server.</string>
+        </property>
+        <property name="text">
+         <string>Do not send OS information</string>
```

Comment:
The problem with this wording is that the option only affects Mumble servers.

The client will (probably?) still send its OS information in HTTP requests, and in statistics to the Mumble project.